src/pages/contact-us/index.tsx

```diff
@@ -5,7 +5,10 @@ import { PageHeader } from '@components/PageHeader'
 const ContactUs = () => (
   <div className="container">
     <div className="grid grid-cols-1 divide-y">
-      <PageHeader title="Contact Us" />
+      <PageHeader
+        title="Contact Us"
+        paths={[]}
```

Comment:
Update interface with `?` to not have empty array in places where paths don't persist

Reply:
Done